demos/mqtt/mqtt_demo_mutual_auth/demo_config.h

```diff
@@ -150,30 +150,35 @@
  * No two clients may use the same client identifier simultaneously.
  */
 #ifndef CLIENT_IDENTIFIER
-    #define CLIENT_IDENTIFIER         "testclient"
+    #define CLIENT_IDENTIFIER    "testclient"
 #endif
 
 /**
  * @brief Size of the network buffer for MQTT packets.
  */
-#define NETWORK_BUFFER_SIZE       ( 1024U )
+#define NETWORK_BUFFER_SIZE              ( 1024U )
 
 /**
  * @brief The name of the operating system or SDK that the application is
  * running a part of.
  */
-#define SDK_NAME                  "aws-iot-device-sdk-embedded-C"
+#define OS_NAME                          "Ubuntu"
```

Comment:
Public docs will have "OS_NAME" and "OS_VERSION", this change is so that customers can follow easily.